crates/ruff_python_ast/src/nodes.rs

```diff
@@ -781,6 +831,92 @@ impl From<ExprDict> for Expr {
     }
 }
 
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, Clone)]
```

It's sometimes useful to have the ability to clone an iterator.